src/lib/svg-optimizer.js

```diff
@@ -21,5 +21,5 @@ const svgoMonochrome = new SVGO({
 
 module.exports = (svgString, monochrome) =>
   (monochrome ? svgoMonochrome : svgo)
-    .optimize(svgString)
+    .optimize(svgString, {})
```

Comment:
Lets do maybe 
```js
module.exports = (svgString, monochrome, path={}) =>
(monochrome ? svgoMonochrome : svgo).optimize(svgString, path)
```

Reply:
Ok, but let's call it `info` instead?

Reply:
Yep, seems logical if SVGO team uses the same name: https://github.com/svg/svgo/blob/master/lib/svgo.js#L24